### PR TITLE
feat:create navbar and pages sample

### PR DESCRIPTION
**Pull request by elzuoc**
new a navbar component, three pages for router sample

### CODE REVIEW

**Comment by alphatero** [comment]
views
-index.js
-otherpages.js

Just named index, please.

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
 /.vscode
+.editorconfig
```
Comment (alphatero) [inline .gitignore:4]: What is the file doing?

File: src/components/NavBar.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            <Link to="/ExampleLearn">Learn Example</Link>
+          </li>
+          <li>
+            <Link to="/ExampleChallenge">Challenge Example</Link>
+          </li>
+        </ul>
+
+        <hr />
+
+        <Routes>
```
Comment (alphatero) [inline src/components/NavBar.js:30]: Navbar just for navigation, router view usually in views or app main.

File: src/components/test.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+console.log();
```
Comment (alphatero) [inline src/components/test.js:1]: Please remove console before push comment.

Comment (alphatero) [reply src/components/test.js:1]: If not, my 助教 will teach you.

**Review by alphatero — COMMENTED** [review]
Please check comment.

File: .gitignore (hunk 1 above)
Comment (elzuoc) [reply .gitignore:4]: sorry, this file is no use.

File: src/components/test.js (hunk 3 above)
Comment (elzuoc) [reply src/components/test.js:1]: ok, and sorry, this file is no use.

File: src/views/index.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function IndexPage() {
+  return (
+
+    <div className="IndexPage">
```
Comment (alphatero) [inline src/views/index.js:6]: Hi, maybe here named home as same as link name is better.

**Review by alphatero — COMMENTED** [review]
Just one issue.

**Review by alphatero — APPROVED** [review]
good job

**Pull request merged**